packages/appstore/src/modules/traders-hub/index.tsx

```diff
@@ -84,6 +85,20 @@ const TradersHub = () => {
             >
                 {can_show_notify && <Notifications />}
                 <div id='traders-hub' className='traders-hub' ref={traders_hub_ref}>
+                    <WalletHeader account_type='demo' />
+                    <WalletHeader currency={'AUD'} jurisdiction={'svg'} />
+                    <WalletHeader currency={'EUR'} jurisdiction={'svg'} is_open_wallet={true} />
+                    <WalletHeader currency={'USD'} jurisdiction={'malta'} />
+                    <WalletHeader currency={'BTC'} jurisdiction={'svg'} />
```

Comment:
you can define actions props to pass array of actions with the proper functionality, type and button, some thing like below:
ex: [
{type: 'transfer, action: () => concole.log('Transfer'), button: {localize('Transfer')}},
{type: 'deposit, action: () => concole.log('Deposit'), button: {localize('Deposit')},
...
]

Reply:
@mahdiyeh-deriv I understand what you mean, but in my opinion in this case it's not a good idea. I've updated PR and let me know where I need to put array `btns` from `WalletHeaderButtons`, inside of `WalletHeader` or outside of it? In this we need to create these arrays (1 for demo and 1 for real) somewhere in the parent component and wrap it in 'useMemo' because `WalletHeader` is inside of React.memo. I'm asking because I'm not sure we will use these buttons wherever instead of `WalletHeaderButtons` component.

Reply:
@sergei-deriv you can pass actions to the walletButton component and instead of `
    const btns: TWalletButton[] = is_demo
        ? [ .....`
 you can loop through the actions(button_group or whatever you like to call it).In my opinio keeping WalletButtons component as a separate component and not merging with the `walletHeader` is making the `walletHeader` more clean and readable, but if you think it's no point in having separate component for the buttons in this case, I do not mind to remove it.

Reply:
@mahdiyeh-deriv My main question is when I form these arrays in parent component (WalletHeadear) it makes parent component not pure too :) I've created util function, please check is it ok or not